src/kryptos/k4/masking_v2.py

```
@@ -38,12 +39,7 @@ def _recalc_cribs(residue_indices: list[int]) -> dict[str, int | None]:
     """
     idx_map = {orig: new for new, orig in enumerate(residue_indices)}
     result: dict[str, int | None] = {}
-    for crib, plain, orig_pos in [
-        ("EAST", "EAST", 22),
-        ("NORTHEAST", "NORTHEAST", 26),
-        ("BERLIN", "BERLIN", 63),
-        ("CLOCK", "CLOCK", 69),
-    ]:
+    for crib, (plain, orig_pos) in K4_CRIBS.items():
```

Pylint (4.0.7)

[warning] Unused variable 'plain'

(W0612)